optuna/study.py

```diff
@@ -607,6 +611,8 @@ def create_study(
             a :class:`~optuna.structs.DuplicatedStudyError` is raised if ``load_if_exists`` is
             set to :obj:`False`.
             Otherwise, the creation of the study is skipped, and the existing one is returned.
+        force_garbage_collection:
+            Flag to force gc.collect() for every trial.
```

Comment:
Please quote `gc.collect()` as follows:
```suggestion
            Flag to force ``gc.collect()`` for every trial.
```